resources/kcp/charts/kyma-environment-broker/templates/authorization-policy.yaml

```diff
@@ -143,6 +143,46 @@ spec:
 ---
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
+metadata:
+  name: istio-events
+  namespace: kcp-system
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        methods:
+        - GET
+        paths:
+        - /events
+    from:
+      - source:
+          requestPrincipals:
+          - {{ tpl .Values.oidc.issuer $ }}/*
+    when:
+    - key: request.auth.claims[groups]
+      values:
+      - {{ .Values.oidc.groups.admin }}
+      - {{ .Values.oidc.groups.operator }}
+  - to:
+    - operation:
+        methods:
+        - GET
+        paths:
+        - /events
+    from:
+    - source:
+        principals:
+{{- with .Values.runtimeAllowedPrincipals }}
+{{ tpl . $ | indent 10 }}
+{{- end }}
```

Comment:
afaiu this is for metrics, currently not used but maybe desired follow-up to add metrics?